tests/e2e/specs/settings/applications-passwords.test.js

```diff
@@ -0,0 +1,103 @@
+import {
+    visitAdminPage,
+    __experimentalRest as rest,
+} from '@wordpress/e2e-test-utils';
+
+async function getResponseForApplicationPassword() {
+    return await rest({
+        method: 'GET',
+        path: '/wp/v2/users/me/application-passwords'
+    });
+}
+
+async function createApplicationPassword(applicationName) {
+    await visitAdminPage('profile.php');
+    await page.waitForSelector('#new_application_password_name');
+    await page.type('#new_application_password_name', `${applicationName}`);
```

Comment:
Nit: unnecessary usage of template string?

```suggestion
    await page.type('#new_application_password_name', applicationName);
```